docs/upstream.md

```diff
@@ -42,23 +42,28 @@ upstream        https://github.com/AlaskaAirlines/auro-checkbox.git (push)
 
 #### Sync upstream
 
-To sync your master brach with your upstream master branch, there are two ways you can do this.
-
-One way is to fetch and merge. This will work if your `master` branch shares the same history as the upstream brach sans merged updates since fork or last sync.
+To sync your `main` branch with the upstream `main` branch, there are two ways you can do this. The preferred method is to fetch and rebase. Please do not merge down.
```

Comment:
This repo still uses a `master` branch. Unless we do the work to rename the branch.